src/web/mjs/connectors/ComicTrail.mjs

```diff
@@ -0,0 +1,52 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ComicTrail extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'comictrail';
+        super.label = 'Comic Trail (コミックトレイル)';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://comic-trail.jp';
+
+        this.queryManga = 'section.section div.columns div.column div > a';
+
+        this.queryChapters = 'section.section > ul.columns > li.column.is-full-mobile.is-half-tablet.chapter-item div.columns.is-mobile.is-gapless.chapter-item__inner';
```

Comment:
Query can be simplified
`'ul.chapters li.chapter-item div.chapter-item__inner'`

Reply:
it can me simplified but that one sure doesn't work

Reply:
I tested it in browser and it worked fine, except it goes one level deeper than the original query and therefore needs also adjustments in the chapter queries, e.g.
`element.child[0].child[3].child[1].textContent` would be `element.child[3].child[1].textContent`